app/src/components/progressCircle/types.ts

```diff
@@ -0,0 +1,7 @@
+import { ProgressBarProps } from "react-aria-components";
```

Comment:
I would make it be derived from https://github.com/Arize-ai/phoenix/blob/8f1cf0b39ab2d606d77da4c740175e3419af8739/app/src/components/types/sizing.ts#L1 for consistency.  While it seems stupid it will help keep props consistent